### PR TITLE
Comments and LogEscape

### PR DESCRIPTION
**Pull request by jamesob**
Just a few little changes made while reviewing. Feel free to discard or change any of the documentation edits.

### CODE REVIEW

File: src/mp/util.cpp [hunk 1]
```diff
@@ -81,7 +81,7 @@ std::string LogEscape(const kj::StringTree& string)
     string.visit([&](const kj::ArrayPtr<const char>& piece) {
         if (result.size() > MAX_SIZE) return;
         for (char c : piece) {
-            if ('c' == '\\') {
```
Comment (ryanofsky) [inline src/mp/util.cpp:84]: 🤦

File: include/mp/proxy-io.h [hunk 2]
```diff
@@ -338,6 +338,13 @@ class Connection
 };
 
 //! Vat id for server side of connection. Required argument to RpcSystem::bootStrap()
+//!
+//! "Vat" is Cap'n Proto nomenclature for a host of various objects that facilitates
+//! bidirectional communication with other vats; it is often but not always 1-1 with
+//! processes. Cap'n Proto doesn't reference clients or servers per se; instead everything
+//! is just a vat.
+//!
+//! See also: https://github.com/capnproto/capnproto/blob/9021f0c722b36cb11e3690b0860939255ebad39c/c%2B%2B/src/capnp/rpc.capnp#L42-L56
```
Comment (ryanofsky) [inline include/mp/proxy-io.h:347]: Further back, it's from E https://en.wikipedia.org/wiki/E_(programming_language)#Computational_model

File: src/mp/util.cpp (hunk 1 above)
Comment (jamesob) [reply src/mp/util.cpp:84]: Surprised there isn't a compiler warning for these kinds of things.

**Comment by ryanofsky** [comment]
Thank you!

**Pull request merged**